### PR TITLE
Created GetEmailAliasesAsync and GetEmailAliases_ValidResponse_ValidUser UT

### PR DESCRIPTION
**Pull request by marek-vysoky**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @marek-vysoky has signed the CLA. Thanks for the pull request!

File: Box.V2/Config/Constants.cs [hunk 1]
```diff
@@ -71,7 +71,8 @@ public static class Constants
         public const string AllFileMetadataPathString = @"{0}/metadata";
         public const string MetadataTemplatesPathString = @"{0}/{1}/schema";
         public const string EnterpriseMetadataTemplatesPathString = @"{0}";
-        public const string TasksPathString = @"{0}/tasks";
+		public const string TasksPathString = @"{0}/tasks";
```
Comment (coolboy) [inline Box.V2/Config/Constants.cs]: spacing, looks like you have tab mixed with space?
Let's use 4 spaces and no tab in our code

File: Box.V2/Config/Constants.cs [hunk 2]
```diff
@@ -95,7 +96,8 @@ public static class Constants
         public const string TypeEnterprise = "enterprise";
         public const string TypeLock = "lock";
         public const string TypeWebhook = "webhook";
-        public const string TypeTask = "task";
+		public const string TypeTask = "task";
```
Comment (coolboy) [inline Box.V2/Config/Constants.cs]: ditto

**Comment by marek-vysoky** [comment]
@coolboy Can you check again?

File: Box.V2.Test/BoxResourceManagerTest.cs [hunk 3]
```diff
@@ -24,8 +24,9 @@ public abstract class BoxResourceManagerTest
         protected AuthRepository _authRepository;
 
         protected Uri _baseUri = new Uri(Constants.BoxApiUriString);
-        protected Uri _FoldersUri = new Uri(Constants.FoldersEndpointString);
+		protected Uri _FoldersUri = new Uri(Constants.FoldersEndpointString);
```
Comment (coolboy) [inline Box.V2.Test/BoxResourceManagerTest.cs]: spacing

File: Box.V2.Test/BoxResourceManagerTest.cs [hunk 4]
```diff
@@ -34,15 +35,14 @@ public BoxResourceManagerTest()
             _handler = new Mock<IRequestHandler>();
             _service = new BoxService(_handler.Object);
             _config = new Mock<IBoxConfig>();
-            _config.SetupGet(x => x.CollaborationsEndpointUri).Returns(new Uri(Constants.CollaborationsEndpointString));
+			_config.SetupGet(x => x.CollaborationsEndpointUri).Returns(new Uri(Constants.CollaborationsEndpointString));
```
Comment (coolboy) [inline Box.V2.Test/BoxResourceManagerTest.cs]: spacing

File: Box.V2.Test/BoxResourceManagerTest.cs [hunk 5]
```diff
@@ -34,15 +35,14 @@ public BoxResourceManagerTest()
...
             _config.SetupGet(x => x.FoldersEndpointUri).Returns(_FoldersUri);
-
             _config.SetupGet(x => x.FilesEndpointUri).Returns(_FilesUri);
+            _config.SetupGet(x => x.UserEndpointUri).Returns(_usersUri);
 
             _authRepository = new AuthRepository(_config.Object, _service, _converter, new OAuthSession("fakeAccessToken", "fakeRefreshToken", 3600, "bearer"));
         }
-
-        public static T CreateInstanceNonPublicConstructor<T>()
+		public static T CreateInstanceNonPublicConstructor<T>()
```
Comment (coolboy) [inline Box.V2.Test/BoxResourceManagerTest.cs]: spacing

File: Box.V2/Converter/BoxItemConverter.cs [hunk 6]
```diff
@@ -53,8 +53,10 @@ protected override BoxEntity Create(Type objectType, JObject jObject)
                         return new BoxUserInvite();
                     case Constants.TypeWebhook:
                         return new BoxWebhook();
-                    case Constants.TypeTask:
+					case Constants.TypeTask:
```
Comment (coolboy) [inline Box.V2/Converter/BoxItemConverter.cs]: spacing

**Comment by marek-vysoky** [comment]
@coolboy Can you check again?

**Pull request merged**